### PR TITLE
Bump externalip-webhook to 1.0.1

### PR DESCRIPTION
**Pull request by cmurphy**
https://github.com/rancher/rancher/issues/35100

### CODE REVIEW

**Review by brendarearden — DISMISSED** [review]
LGTM

File: charts/rancher-external-ip-webhook/rancher-external-ip-webhook/100.0.1+up1.0.1/Chart.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ annotations:
   catalog.cattle.io/release-name: rancher-external-ip-webhook
   catalog.cattle.io/ui-component: rancher-external-ip-webhook
```
Comment (aiyengar2) [inline charts/rancher-external-ip-webhook/rancher-external-ip-webhook/100.0.1+up1.0.1/Chart.yaml:8]: needs rancher-version and upstream-version Chart annotations, will also ned to update the index.yaml for the new annotations after that.

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: charts/rancher-external-ip-webhook/rancher-external-ip-webhook/100.0.1+up1.0.1/Chart.yaml (hunk 1 above)
Comment (cmurphy) [reply charts/rancher-external-ip-webhook/rancher-external-ip-webhook/100.0.1+up1.0.1/Chart.yaml:9]: done

**Review by brendarearden — APPROVED** [review]
looks good!

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**